xds/src/main/java/io/grpc/xds/XdsNameResolver.java

```diff
@@ -123,6 +124,7 @@ final class XdsNameResolver extends NameResolver {
   // put()/remove() must be called in SyncContext, and get() can be called in any thread.
   private final ConcurrentMap<String, ClusterRefState> clusterRefs = new ConcurrentHashMap<>();
   private final ConfigSelector configSelector = new ConfigSelector();
+  private final long randomChannelId = ThreadLocalRandom.current().nextLong();
```

Comment:
nit: we can use [ThreadSafeRandom](https://github.com/grpc/grpc-java/blob/d4fa0ecc07495097453b0a2848765f076b9e714c/xds/src/main/java/io/grpc/xds/ThreadSafeRandom.java#L23), isn't it?